nchs_mortality/.pylintrc

```diff
@@ -4,6 +4,7 @@
 disable=logging-format-interpolation,
     too-many-locals,
     too-many-arguments,
+    fixme,
```

Comment:
Added this for a TODO I left in the code. Let me know if this is something we'd want to keep consistent across all indicators and I can add it to others.